prac_06/programming_language.py

```diff
@@ -11,10 +11,7 @@ def __init__(self, name, typing, reflection, year):
         self.year = year
 
     def is_dynamic(self):
-        if self.typing == "Dynamic":
-            return True
-        else:
-            return False
+        return self.typing == "Dynamic"
```

Comment:
Will never return this value as the code is written (else will prevent "Dynamic" being returned). is_dynamic function should only return Boolean. This was most likely left here unintentionally. Perfect code otherwise